user/ip-addresses.md

```diff
@@ -0,0 +1,15 @@
+---
+title: IP Addresses
+layout: en
+permalink: /user/ip-addresses/
+---
+
+For some it might be necessary to know the public IP addresses of build machines in order to whitelist them to access some internal resources. We run builds in a variety of different infrastructures, so which one you use determines which IP range you need to whitelist.
+
+For jobs running on the container-based infrastructure, the ranges are `54.172.141.90/32` and `52.3.133.20/32` for travis-ci.com (private repositories), and `52.0.240.122/32` and `52.22.60.255/32` for travis-ci.org (public repositories).
```

Comment:
And instead of running text I'd present these IP ranges as either a table or or a list for ease of lookup:

Infrastructure | IP range
------------ | -------------
container based | xxxx-yyyy

OR
* Container based: xxxx-yyyy
* Private Repositories: xxxxx-yyyy